RabbitMQ.Stream.Client/StreamSystem.cs

```diff
@@ -15,6 +15,7 @@ public record StreamSystemConfig : INamedEntity
         public string UserName { get; set; } = "guest";
         public string Password { get; set; } = "guest";
         public string VirtualHost { get; set; } = "/";
+        public uint Heartbeat { get; set; } = 60;
```

Comment:
Should this be changed to `TimeSpan` so it's more obvious what's passed in? Then `HeartBeatHandler` wouldn't need to have a variable named `timerIntervalSeconds`.

Reply:
Good point, but `Heartbeat` is in seconds I would  leave it in seconds

Reply:
You could extract `TotalSeconds` from that `TimeSpan` internally but eh, it works.